query-algebrizer/src/errors.rs

```diff
@@ -31,7 +31,7 @@ error_chain! {
         }
 
         UnboundVariable(name: PlainSymbol) {
-            description("unbound variable in function call")
```

Comment:
I expect this to happen a lot -- it will happen in function applications, etc.  Why don't we include some `context` argument to ease this?